zeppelin-web/src/app/credential/credential.controller.js

```diff
@@ -12,49 +12,60 @@
  * limitations under the License.
  */
 
-angular.module('zeppelinWebApp').controller('CredentialCtrl', CredentialCtrl)
+angular.module('zeppelinWebApp').controller('CredentialCtrl', CredentialController)
 
-function CredentialCtrl ($scope, $rootScope, $http, baseUrlSrv, ngToast) {
+function CredentialController($scope, $http, baseUrlSrv, ngToast) {
   'ngInject'
 
-  $scope._ = _
   ngToast.dismiss()
 
   $scope.credentialInfo = []
   $scope.showAddNewCredentialInfo = false
   $scope.availableInterpreters = []
 
+  $scope.entity = ''
+  $scope.password = ''
+  $scope.username = ''
+
+  $scope.hasCredential = () => {
+    return Array.isArray($scope.credentialInfo) && $scope.credentialInfo.length
+  }
+
   let getCredentialInfo = function () {
     $http.get(baseUrlSrv.getRestApiBase() + '/credential')
-    .success(function (data, status, headers, config) {
-      $scope.credentialInfo = _.map(data.body.userCredentials, function (value, prop) {
-        return {entity: prop, password: value.password, username: value.username}
+      .success(function (data, status, headers, config) {
```

Comment:
Thanks for the information, we can solve that issue in [ZEPPELIN-2788](https://issues.apache.org/jira/browse/ZEPPELIN-2788)